### ISSUE TITLE
Cheat sheet

### ISSUE THREAD

**Issue by MagMueller** [lorenzwalthert/comp_stats_summary#2]
Did you write a cheat sheet for this exam?

**Comment by lorenzwalthert** [comment]
yes, but I can't find it anymore... Also, I think the content of the course and exam are not the same anymore since it's not Martin Maechler anymore who teaches.
